resources/assets/typescript/utils.ts

```diff
@@ -0,0 +1,7 @@
+export default class Utils {
+    forEach(array, callback, scope) {
```

Comment:
Should this not be in a function? Maybe I've just never come across this syntax before...

Reply:
It's just defining a class method, I've made it a public method so it's a little more clear.

Reply:
Nice, thanks.